packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -20,15 +20,23 @@ interface IElectionModule {
 
     function setSeatCount(uint seats) external;
 
-    function setEpochDuration(uint duration) external;
+    function setEpochDuration(uint64 duration) external;
```

Comment:
changed to uint64 to prevent overflows when operating with it

Reply:
Does this mean that the duration of the current epoch can be changed on the run? Won't this complicate logic a lot and introduce fragility?

Reply:
Yes, here I was respecting the original spec. But it makes sense to keep it fixed